src/main/java/com/ldtteam/domumornamentum/block/types/PillarShapeType.java

```diff
@@ -22,4 +25,11 @@
     {
         return this.name();
```

Comment:
instead of the new method, just return this.name without parenthesis here.

Reply:
i don't remember making that change, but its fixed